README.md
# ssbwear
For our beloved game, SLINGSHOT BRAVES :sparkling_heart:

:zap: <a href='https://diewland.github.io/ssbwear/'>LIVE</a>

## Special Thanks
* Slingshot Braves TH 
* English SSB Line Group
* Angelic 2 Braves


db/changelog.txt
@riryoku on 01/11/2015 09:02 ICT
- All remaining slots of "Halloween Trickster Hat" have been unlocked.

@riryoku on 30/10/2015 23:35 ICT
- All remaining slots of "Halloween Night-Reaper", "Halloween Trickster" and "Halloween Tricker Robe" have been unlocked.

@diewland on 30/10/2015 17:00 ICT
- Added "Halloween Trickster" gacha set

@riryoku on 17/10/2015 00:13 ICT
- Corrected the square element of "Shadow Knight's Helm".

@riryoku on 16/10/2015 14:57 ICT
- All remaining slots of "The Hornet Queen" have been unlocked.
- Updated cost of "The Hornet Queen".
- Corrected string in "slot.js".

@riryoku on 15/10/2015 23:33 ICT
- The 6th slot of "Lethal Rod of Ordnance" has been unlocked.

@riryoku on 14/10/2015 02:30 ICT
- Updated cost and 6th slot of "Hat of the Braves".

@diewland on 13/10/2015 01:31 ICT
- Extra slots related on bravanization

@riryoku on 11/10/2015 22:54 ICT
- The last slot of "Shadow Knight's Mail" has been unlocked.

@riryoku on 11/10/2015 17:40 ICT
- Uploaded these pictures.
1) "Dragon Ice Inferno" is "2005"
2) "Flare Cyclone" is "1112"
3) "Tempest of Heaven" is "3049"
4) "Blaze of Darkness" is "3018"
5) "Lightdrake's Claw Absolut-Justitia" is "3070"
6) "Queen Ether of Sparklight" is "3056"

@riryoku on 11/10/2015 15:35 ICT
- The last slot of "Walpurgis Robe" has been unlocked.

@riryoku on 11/10/2015 12:22 ICT
- The 6th slot of "Shadow Knight's Mail" has been unlocked.

@diewland on 10/10/2015 19:27 ICT
- Simulator data will not lose when refresh or exit

@riryoku on 10/10/2015 12:20 ICT
- All remaining slots of "Shadow Knight's Helm" have been unlocked.

@diewland on 09/10/2015 22:49 ICT
- "Shadow Knight" photos have been added.

@riryoku on 09/10/2015 14:27 ICT
- All new Gacha items "Shadow Knight" have been added.
Index
1) "Shadow's Wrath" is "3165".
2) "Shadow Piercer" is "3166".
3) "Shadow Knight's Helm" and "Shadow Knight's Mail" are "3113".
- All remaining slots of "Shadow's Wrath" have been unlocked. 
- All pics will be uploaded by @diewland.

@riryoku on 09/10/2015 00:47 ICT
- The 6th and 7th of "Jyr Yong Blade" have been unlocked.
- Fixed some errors in DB.

@riryoku on 08/10/2015 12:20 ICT
- Added "Lost Key to Atlantica" the weapon from ranking event with completely details.

@diewland on 06/10/2015 21:03 ICT
- Added Oriax Star

@riryoku on 06/10/2015 18:24 ICT
- The brave skill of "Lucifer's Fallen Mail" and "High Roller's Blackjack Suit" have been added.

@riryoku on 06/10/2015 09:24 ICT
- The 7th slot of "Medjed's Crown of the Dead" and "Medjed's Robe of the Dead" have been unlocked.

@diewland on 06/10/2015 00:25 ICT
- Added The Hornet Queen sword

@riryoku on 04/10/2015 23:45 ICT
- Corrected the max brave skill of "Apollion Epic Mail".
- The 6th slot of "Lamia's Vengeance" has been unlocked.
- Corrected the cost of "Lamia's Vengeance".

@riryoku on 03/10/2015 14:28 ICT
- The 7th slot of "Epoch Chronos Lance" and "Walpurgis Hat" have been unlocked.

@diewland on 03/10/2015 01:23 ICT
- "Epoch Chronos" photos have been uploaded

@riryoku on 02/10/2015 16:56 ICT
- The 6th slot of "Epoch Chronos Lance" has been unlocked.

@riryoku on 02/10/2015 14:29 ICT
- All new Gacha items "Epoch Chronos" have been added
- The 6th and 7th of "Epoch Chronos Rod", "Epoch Chronos Helm" and "Epoch Chronos Mail" have been unlocked.
- Index of all new Gacha 
1) "3110" for costume
2) "3150" for Lance
3) "3151" for Rod
Wait @diewland for upload them all.

@riryoku on 02/10/2015 08:13 ICT
- Added the last year Halloween outfit "All Hallows' Hat/Headband" and "All Hallows' Garb" with initial details.
- Index of Halloween outfit is no longer available on my device. We will upload when found them all.

@riryoku on 01/10/2015 08:05 ICT
- The 7th slot of "Spear of Valhalla" has been added.
- Corrected some values.

@riryoku on 30/09/2015 14:10 ICT
- Added obsoleted boss weapons as follow.
1) Dragon Ice Inferno
2) Blaze of Darkness
3) Flare Cyclone
4) Tempest of Heaven
5) Lightdrake's Claw Absolut-Justitia
6) Queen Ether of Sparklight
- Wait for SS picture of all from @diewland.

@riryoku on 30/09/2015 09:06 ICT
- Added some obsoleted event gears as below.
1) "Dendron Hat" + "Dendron Garb" 
2) "Helmet of Gloria" + "Armor of Sanctus" and "Armor of Sanctus+"
3) "Divine Helm of Renatus" + "Divine Armor of Renatus"
4) "Helm of Holy Light" + "Harness of Holy Light"
5) "Cobalt Crown" + "Cerulean Sea Mail"
6) "Pure Luminous Pendragon" + "Pure Photon Arts"
7)  "Helmet of Cavaliere" + "Armor of Equitatus"
- Some obsoleted gear pictures are no longer available on my device due to they were dismantled. please wait to trial and error. We will upload them when we found.

@riryoku on 30/09/2015 08:02 ICT
- The 7th slot of "Lucifer's Fallen Mail" and "Samsara Lance Longinus" have been added.

@diewland on 29/09/2015 22:45 ICT
- Upload "Hollow Edge Albion X" photo
- Added Vapula's Wrath, Legendary Item
- Added Lost Key to Atlantica, King of Quest 3 Item

@riryoku on 29/09/2015 14:00 ICT
- Corrected some values.

@riryoku on 28/09/2015 11:17 ICT
- Added current event weapon "Hollow Edge Albion X" with 6th slot.
- Wait @diewland for upload picture code "1505" for "Hollow Edge Albion" series.

@riryoku on 27/09/2015 04:58 ICT
- The 7th of the both "Brave Valor" head and body have been added by based on the "Noble Valor".

@diewland on 25/09/2015 23:15 ICT
- Upload pictures of Limited Dark-Light Equipments

@riryoku on 25/09/2015 16:11 ICT
- The 6th and 7th slots of "Lamia's Agony" have been added.

@riryoku on 25/09/2015 13:54 ICT
- Added current event suit "Orocoth Ale" with completely details.
- Wait @diewland for upload picture code "9028".
- Added new limited entries "Lamia's Agony" and "Lamia's Vengeance" with initial details.
- Wait @diewland for upload both pictures of weapon code "3169" for "Lamia's Agony" and "3168" for "Lamia's Vengeance".

@riryoku on 25/09/2015 12:49 ICT
- Added new limited entries "Bloodlust Vow" and "Blood of Vengeance" with completely details.

db/db-wood.js
var WOODS = [
    {
        code: 'AIN_CHA',    name: "Ainsel Chapeau",
        type: HEAD,         material: WOOD,
        score: 4171,        slots: 'BAT_5-SKL_5-ATK_5-KIT-CIR|DAI',
        cost: 35,
    },
    {
        code: 'AIN_COA',    name: "Ainsel Coat",
        type: ARMOR,        material: WOOD,
        score: 2350,        slots: 'BAT_5-HEA_5-HP_5-REC-KIT',
        cost: 35,
    },
    {
        code: 'HCR_HLM',    name: "Hachiryu's Helmet",
        type: HEAD,         material: WOOD,
        score: 4201,        slots: 'KTS_4-HEA_5-TRI-CIR-TRI|CIR',
        cost: 35,
    },
    {
        code: 'HCR_AMR',    name: "Hachiryu's Armor",
        type: ARMOR,        material: WOOD,
        score: 2288,        slots: 'KTS_4-HAS_5-QUI_5-KIT-CIR|TRI',
        cost: 35,
    },
    {
        code: 'SSN_AMR',    name: "Armor of Susano-o",
        type: ARMOR,        material: WOOD,
        score: 2563,        slots: 'LSG_5-HAS_9-SKL_10-TRI-CIR|CIR',
        limited: true,      cost: 40,
    },
    {
        code: 'YTK_HLM',    name: "Helmet of Yata no Kagami",
        type: HEAD,         material: WOOD,
        score: 4614,        slots: 'LSG_5-HEA_9-HP_10-TRI-TRI|TRI',
        limited: true,      cost: 40,
    },
    {
        code: 'BSM_STT',    name: "Balsamina Striata",
        type: HEAD,         material: WOOD,
        score: 4239,        slots: 'DVR_5-QUI_9-HEA_9-CIR-TRI|CIR',
        cost: 37,
    },
    {
        code: 'BSM_EPB',    name: "Balsamina Euphorbia",
        type: ARMOR,        material: WOOD,
        score: 2360,        slots: 'DVR_5-CMB_9-SKL_9-DAI-CIR|CIR',
        cost: 37,
    },
    {
        code: 'CLT_HLM',    name: "Celestial Helm",
        type: HEAD,         material: WOOD,
        score: 4305,        slots: 'LSG_5-HAS_9-KOH_5-TRI-KIT|TRI',
        cost: 50,
    },
    {
        code: 'CLT_MAI',    name: "Celestial Mail",
        type: ARMOR,        material: WOOD,
        score: 2432,        slots: 'LSG_5-QUI_9-HEA_10-REC-CIR|TRI',
        cost: 50,           skill: 'CHO_MTO_1',
    },
    {
        code: 'DVN_EFS',    name: "Divine Efflorescence",
        type: HEAD,         material: WOOD,
        score: 4350,        slots: 'HDB_5-QUI_10-SRS_4-TRI-KIT|CIR',
        cost: 37,
    },
    {
        code: 'PSP_BLS',    name: "Persephone's Blessing",
        type: ARMOR,        material: WOOD,
        score: 2480,        slots: 'DVR_5-HEA_10-SRS_4-TRI-DAI|TRI',
        cost: 37,
    },
    {
        rcode: 3026,        name: "Holy Blue Dragon Helmet",
        type: HEAD,         material: WOOD,
        score: 3864,        slots: 'BAT_4-SKL_5-SKA_3-KIT-REC|CIR',
        cost: 34,
    },
    {
        rcode: 3026,        name: "Holy Blue Dragon Plate",
        type: ARMOR,        material: WOOD,
        score: 2139,        slots: 'BAT_4-HAS_5-KIT-KIT-CIR|DAI',
        cost: 34,
    },
    {
        rcode: 3048,        name: "Divine Helm of Renatus+",
        type: HEAD,         material: WOOD,
        score: 4019,        slots: 'BAT_4-WNDD_5-HEA_10-CIR-TRI',
        cost: 37,           evt: true,
    },
    {
        rcode: 3048,        name: "Divine Armor of Renatus+",
        type: ARMOR,        material: WOOD,
        score: 2357,        slots: 'BAT_4-WNDD_5-SKL_10-CIR-DAI',
        cost: 37,           evt: true,
    },
    {
        rcode: 3048,        name: "Divine Helm of Renatus",
        type: HEAD,         material: WOOD,
        score: 3140,        slots: 'BAT_3-WNDD_3-HEA_3-CIR-TRI',
        cost: 31,           evt: true,
    },
    {
        rcode: 3048,        name: "Divine Armor of Renatus",
        type: ARMOR,        material: WOOD,
        score: 1753,        slots: 'BAT_3-WNDD_3-SKL_4-CIR-DAI',
        cost: 31,           evt: true,
    },
    {
        rcode: 9999,        name: "Dendron Hat",
        type: HEAD,         material: WOOD,
        score: 3140,        slots: 'KTS_3-SKA_3-SKL_3-CIR-TRI',
        cost: 0,           evt: true,
    },
    {
        rcode: 9999,        name: "Dendron Garb",
        type: ARMOR,        material: WOOD,
        score: 2003,        slots: 'KTS_3-HP_5-HEA_3-CIR-DAI',
        cost: 0,           evt: true,
    },
    {
        rcode: 3083,        name: "Quick Helm of Idaten",
        type: HEAD,         material: WOOD,
        score: 4830,        slots: 'AVG_6-HEA_10-KOH_5-TRI-REC|DAI-CIR',
        cost: 50,           limited: true,
    },
    {
        rcode: 3083,        name: "Gallant Mail of Idaten",
        type: ARMOR,        material: WOOD,
        score: 2780,        slots: 'AVG_6-SPB_5-KOH_5-DAI-TRI|CIR-REC',
        cost: 50,           limited: true,
    },
    {
        rcode: 3002,        name: "Headband of Pure Chastity",
        type: HEAD,         material: WOOD,
        score: 4570,        slots: 'HDB_5-SUR_5-SKA_10-EFR_4-TRI|REC-REC',
        cost: 42,
    },
    {
        rcode: 3002,        name: "Dress of Pure Chastity",
        type: ARMOR,        material: WOOD,
        score: 2570,        slots: 'HDB_5-FUR_5-HAS_10-DMH_4-CIR|TRI',
        cost: 42,
    },
    {
        rcode: 3085,        name: "Pop Star's Headress",
        type: HEAD,         material: WOOD,
        score: 4450,        slots: 'TOP_4-ELF_5-HEA_10-CIR-CIR|TRI',
        cost: 42,           evt: true,
    },
    {
        rcode: 3085,        name: "Pop Star's Costume",
        type: ARMOR,        material: WOOD,
        score: 2430,        slots: 'TOP_4-DSM_5-HEA_10-CIR-TRI|CIR',
        cost: 42,           evt: true,
    },
    {
        rcode: 3096,        name: "Elven Eithel Chapeau",
        type: HEAD,         material: WOOD,
        score: 4870,        slots: 'AVG_5-TMV_6-HEA_10-CIR-TRI|DAI-TRI',
        cost: 42,
    },
    {
        rcode: 3096,        name: "Elven Eithel Cuirass",
        type: ARMOR,        material: WOOD,
        score: 2770,        slots: 'AVG_5-VSP_6-SOG_5-CIR-TRI|REC-REC',
        cost: 42,
    },
    {
        rcode: 3094,        name: "Galactical Cosmos Cap",
        type: HEAD,         material: WOOD,
        score: 4380,        slots: 'KTS_5-HVH_5-WLW_4-DAI-CIR|CIR',
        cost: 42,           evt: true,
    },
    {
        rcode: 3094,        name: "Galactical Cosmos Robe",
        type: ARMOR,        material: WOOD,
        score: 2480,        slots: 'KTS_5-MGR_4-ELF_5-CIR-DAI|TRI',
        cost: 42,           evt: true,
    },
    {
        rcode: 3032,        name: "Hat in Full Bloom",
        type: HEAD,         material: WOOD,
        score: 4570,        slots: 'HDB_5-SPB_5-SCT_3-CIR-CIR|TRI',
        cost: 42,           evt: true,
    },
    {
        rcode: 3032,        name: "Summer in Full Bloom",
        type: ARMOR,        material: WOOD,
        score: 2570,        slots: 'HDB_5-NBR_5-KOH_4-TRI-KIT',
        cost: 42,            evt: true,
    },
    {
        rcode: 3093,        name: "Gallivant Viage Hat",
        type: HEAD,         material: WOOD,
        score: 4950,        slots: 'IVT_5-HEA_10-MGR_6-CIR-TRI|TRI-DAI',
        cost: 45,
    },
    {
        rcode: 3093,        name: "Gallivant Viage Coat",
        type: ARMOR,        material: WOOD,
        score: 2860,        slots: 'IVT_5-SPB_6-TMV_5-CIR-TRI|TRI-CIR',
        cost: 45,           skill: 'FRZ_OBT_3',
    },
    {
        rcode: 3103,        name: "Hat of the Stargazer",
        type: HEAD,         material: WOOD,
        score: 5120,        slots: 'LSG_6-WLW_6-HWL_5-CIR-TRI|REC-TRI',
        cost: 45,           limited: true,
    },
    {
        rcode: 3103,        name: "Robe of the Stargazer",
        type: ARMOR,        material: WOOD,
        score: 3150,        slots: 'LSG_6-SPB_6-HWL_5-TRI-TRI|CIR-DAI',
        cost: 45,           limited: true,
        skill: 'PTN_MTO_3',
    },
    {
        rcode: 9027,        name: "Highland Festive Cap",
        type: HEAD,         material: WOOD,
        score: 4800,        slots: 'HDB_5-SPB_5-VSP_5-TRI-CIR|DAI-DAI',
        cost: 45,            evt: true,
    },
    {
        rcode: 9027,        name: "Highland Festive Dress",
        type: ARMOR,        material: WOOD,
        score: 2800,        slots: 'HDB_5-SPB_5-KOH_5-TRI-KIT|TRI',
        cost: 45,            evt: true,
        skill: 'WND_SHF_3',
    },
    {
        rcode: 9999,        name: "All Hallows' Headband",
        type: HEAD,         material: WOOD,
        score: 2814,        slots: 'BAT_3-SKL_3-SKA_3-KIT-REC',
        cost: 28,            evt: true,
        name2: "All Hallows' Hat",
    },
    {
        rcode: 9999,        name: "All Hallows' Garb",
        type: ARMOR,        material: WOOD,
        score: 1323,        slots: 'BAT_3-HAS_3-KIT-KIT-REC',
        cost: 28,            evt: true,
    },
// 2015/10/02
    {
        rcode: 3110,        name: "Epoch Chrono Helm",
        type: HEAD,         material: WOOD,
        score: 5100,        slots: 'DVS_5-EFR_6-HWL_5-HEA_10-DAI|REC-TRI',
        cost: 45,
    },
    {
        rcode: 3110,        name: "Epoch Chrono Mail",
        type: ARMOR,        material: WOOD,
        score: 3120,        slots: 'DVS_5-HWL_6-HWL_5-QUI_10-KIT|CIR-REC',
        cost: 45,           skill: 'RZR_WHW_3',
    },
];
